test/studies/isx/isx-no-return.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: total =, bucket count =, bucket offset =, bucketize =, exchange =, count keys =
+files: isx-no-return.dat
```

Comment:
OK, I think I'll change it for all 3 after merging this to separate the concerns.  But I'm thinking I'm going to sit on this until tomorrow before merging it to avoid rushing it in and avoid the risk of breaking Ben's optimization runs.